src/index.js

```diff
@@ -449,11 +449,15 @@ function createConfig(options, entry, format, writeMeta) {
 							sourcemap: true,
 							output: { comments: false },
 							compress: {
+								passes: 10,
+								hoist_funs: true,
```

Comment:
hm, funny that it makes sense - would assume that this might make things worse, unhoisted should have generally speaking shorter identifiers than hoisted ones

Reply:
I can vouch for the performance improvements 👍